README.md

```diff
@@ -56,8 +56,6 @@ Check on [contribution_guide_usability_testing.md](docs/contribution_guide_usabi
 
 #### Tips for specific circumstances
 
-* Docker: Docker was previously used for development of Houdini. 
-See [docker.md](docs/docker.md) for more info.
 * Mac: Mac dev setup may require some unique configuration. 
 See [mac_getting_started.md](docs/mac_getting_started.md) for more info.
```

Comment:
I think we should remove this entire session or update it, this `mac_getting_started.md` does not exist anymore.